packages/create-plugin/templates/common/_package.json

```diff
@@ -76,7 +76,7 @@
     "@grafana/runtime": "^11.5.2",
     "@grafana/ui": "^11.5.2",
     "@grafana/schema": "^11.5.2",{{#if_eq pluginType "scenesapp"}}
-    "@grafana/scenes": "^5.41.3",{{/if_eq}}
+    "@grafana/scenes": "^6.4.0",{{/if_eq}}
```

Comment:
@leventebalogh this bump forces react-router v6 for scenes, right? Is that gonna cause issues at all? We have [some logic here](https://github.com/grafana/plugin-tools/blob/main/packages/create-plugin/src/utils/utils.templates.ts#L104-L106) that decides the version of react-router to scaffold / update with. 🤔

Reply:
Yeah, I think (as we discussed in a different chat) we should exclude scenes from these updates probably. Scenes v6.x is depending on react-router v6, which plugins that update might don't have. Probably the best is to handle it with a migration to be honest!

(There's also a PR for updating the scenes template: https://github.com/grafana/plugin-tools/pull/1422/files#diff-a56a7e2a8f865015afd4eb379bc0e0ebf6053eeede03d81017b4c9699fe2c772)